test/passing/open_auto.ml

```diff
@@ -0,0 +1,19 @@
+let () =
+  ( (let open Term in
```

Comment:
Too many parenthesis here as well I guess

Reply:
Doesn't look like too much. Does it ?

Reply:
I would have expected something like:

```
let () =
  ( let open Term in
     ...
      $ setup_logs )
  , Term.info "phase1" ~doc ~sdocs:Manpage.s_common_options ~exits ~man )
```

e.g. one pair of parenthesis less.

Reply:
This does not parse the same.

Reply:
woops, indeed :-) sorry for the noise..